package.json

```diff
@@ -36,17 +36,9 @@
   ],
   "license": "SEE LICENSE IN LICENSE.md",
   "lint-staged": {
-    "*.md": [
-      "npm run textlint",
-      "git diff --exit-code --quiet"
-    ],
-    "*.{js,css}": [
-      "npm run format"
-    ],
-    "package.json": [
-      "npx fixpack",
-      "git diff --exit-code --quiet"
-    ]
+    "*.md": "textlint --dry-run",
+    "*.+(css|js)": "prettier --check",
+    "package.json": "npx fixpack --dryRun"
```

Comment:
```
.husky/pre-commit: line 4: npm: command not found
.husky/pre-commit: line 4: /lint-staged: No such file or directory
husky - pre-commit hook exited with code 1 (error)
```

コミットしようとしたらそもそもcommandがないって怒られてしまいまして。
`npm bin` しようとする時ってpackage.json側に `bin` 加えなければダメなんでしょうか？

Reply:
おや、私の環境だと動作はしました。`npm`がないと言われるのはなんとも…… `npm ci` をしてもらっても多分ダメですよね？

```console
$ git ci -m test
  ✔ Preparing...
  ❯ Running tasks...
    ↓ Running tasks for *.md [skipped]
      → No staged files match *.md
    ↓ Running tasks for *.+(css|js) [skipped]
      → No staged files match *.+(css|js)
    ❯ Running tasks for package.json
      ✖ npx fixpack --dryRun
  ↓ Applying modifications... [skipped]
    → Skipped because of errors from tasks.
  ✔ Reverting to original state...
  ✔ Cleaning up...



✖ npx found some errors. Please fix them and try committing again.
missing main
package.json not fixed!
npx: 18個のパッケージを3.797秒でインストールしました。
husky - pre-commit hook exited with code 1 (error)
```

Reply:
やはりそうですよね。ちょっと調べてみます。
僕の環境だけかもしれないので。

Reply:
修正いただいたhttps://github.com/openameba/a11y-guidelines/pull/196の方では指定いただいたコマンドをターミナルから実行していたので大丈夫でした。

今回はSourceTreeを使用して確認してたのですが、調べた感じ.haskyの設定ファイルなどが読み込めなさそうでターミナルからGUIアプリを起動する必要がありそうでした。

https://community.atlassian.com/t5/Bitbucket-questions/SourceTree-Hook-failing-because-paths-don-t-seem-to-be-set/qaq-p/274792

Reply:
package.json, .md, .css, .js含めコミット時にエラー検知してくれること確認できました。
諸々ご対応ありがとうございますー。

記事見た感じなのですがGUIアプリ使っている人は同様の問題が発生しそうな気がしているのですが今後、新しい作業する方がいたら問題になったりしそうですかね？

Reply:
GUIクライアントを使っている場合は問題になるかもしれないですね。